README.md

```diff
@@ -1,2 +1,6399 @@
-### Connector Fortinet FortiManager
-Repository to maintain Fortimanager connector documentation.
+<h2>About the connector</h2>
+
+<p>Fortinet FortiManager provides easy centralized configuration, policy-based provisioning, update management, and end-to-end network monitoring for your Fortinet-installed environment.</p>
+
+<p>This document provides information about the Fortinet FortiManager Connector, which facilitates automated interactions with your Fortinet FortiManager server using FortiSOAR&trade; playbooks. Add the Fortinet FortiManager connector, as a step in FortiSOAR&trade; playbooks and perform automated operations such as retrieving a list of all devices configured on the Fortinet FortiManager server, creating and updating incidents on the Fortinet FortiManager server, and retrieving a list of all incidents from the Fortinet FortiManager server.</p>
+
+<p>You can use FortiSOAR&trade;'s Data Ingestion Wizard to easily ingest data into FortiSOAR&trade; by pulling incidents from Fortinet FortiManager. For more information, see the <a href="#dataIngestion">Data Ingestion Support</a> topic.</p>
+
+<h3>Version information</h3>
+
+<p>Connector Version: 4.0.0</p>
+
+<p>FortiSOAR&trade; Version Tested on: 7.4.3-3294</p>
+
+<p>Fortinet FortiManager Version Tested on: FortiManager VM64-KVM-V7.2.2 build6198</p>
+
+<p>Authored By: Fortinet</p>
+
+<p>Certified: Yes</p>
+
+<h3>Release Notes for version 4.0.0</h3>
+
+<p>The following changes have been made to the Fortinet FortiManager Connector in version 4.0.0:</p>
+
+<ul>
+    <li>Added a new config parameter named <strong>Workspace Mode Enabled</strong> to identify whether the workspace mode is enabled or disabled on FortiManager.</li>
```

Comment:
Please add these in release notes

- Opertaions: "Get Installation Policy Package Status" and "Get User Groups List" have updated output schema.
- Renamed parameter "firewall policy name" to "firewall/security policy name" in "List ADOM Firewall Policies" action.
- In Operations: "Create Firewall Policy" and "Update Firewall Policy" parameters: "Inspection Mode" and "Schedule Timeout" is now moved under parameter "NGFW Mode".

Reply:
Done